src/super_gradients/training/qat_trainer/qat_trainer.py

```diff
@@ -147,6 +172,8 @@ def train_from_config(cls, cfg: Union[DictConfig, dict]) -> Tuple[nn.Module, Tup
             model.train()
             recipe_logged_cfg = {"recipe_config": OmegaConf.to_container(cfg, resolve=True)}
             trainer = Trainer(experiment_name=cfg.experiment_name, ckpt_root_dir=get_param(cfg, "ckpt_root_dir", default_val=None))
+            torch.cuda.empty_cache()
```

Comment:
Why do we need it here?

Reply:
To clean after the previous trainer object that did validation of a PTQ model. 
That's a part of a bigger issue with running Trainer.train after Trainer.test (because init of everything is different)